config/systemjs.config.js

```diff
@@ -108,21 +112,17 @@ System.config({
       "vm-browserify": "npm:vm-browserify@0.0.4"
     },
     "npm:angular2@2.0.0-beta.12": {
-      "crypto": "github:jspm/nodelibs-crypto@0.1.0",
-      "es6-shim": "npm:es6-shim@0.35.0",
-      "process": "github:jspm/nodelibs-process@0.1.2",
-      "reflect-metadata": "npm:reflect-metadata@0.1.2",
+      "reflect-metadata": "npm:reflect-metadata@0.1.3",
       "rxjs": "npm:rxjs@5.0.0-beta.2",
-      "zone.js": "npm:zone.js@0.6.6"
+      "zone.js": "npm:zone.js@0.5.15"
```

Comment:
agreed, that is odd, and I didn't do that explicitly, let me look at that...

On Mon, Mar 28, 2016 at 12:06 PM, cavis <notifications@github.com> wrote:

> In config/systemjs.config.js
> <https://github.com/PRX/publish.prx.org/pull/5#discussion_r57589263>:
>
> >        "rxjs": "npm:rxjs@5.0.0-beta.2",
> > -      "zone.js": "npm:zone.js@0.6.6"
> > +      "zone.js": "npm:zone.js@0.5.15"
>
> Hmm... this downgrade is a bit worrying. Is hot-reloading still working
> for you? I thought I needed 0.6.6 to get it working. But I'll pull down
> this branch and re-check.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/PRX/publish.prx.org/pull/5/files/63ad05fe8561d4449a749475e899197984dcd38a#r57589263>
>